py3status/modules/player_control.py

```diff
@@ -10,6 +10,7 @@
 Configuration parameters:
     cache_timeout: how often to update in seconds (default 10)
     debug: enable verbose logging (bool) (default False)
+    format: string to print (default "{icon}")
```

Comment:
`string to print` feels wrong, we should have a standard text `format of output`?